saml.Test/saml.Test.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
1. This saml.Test.sln - please change name to Saml.sln because we need a solution for the project anyway, and it is usual to have the tests in the main solution.

2. Move this solution file to parent directory so it is easy to find, and check that it opens OK (you might have to edit the text of the .sln file to fix the directory locations)